### ISSUE TITLE
migrate: removeJsonLoaderTransform undefined does not match field "value"

### ISSUE THREAD

**Issue by timoxley** [webpack/webpack-cli#157]
Trying to migrate a mildly edited, ejected, create-react-app config, complains on `removeJsonLoaderTransform`. I have no json loader, also still fails when I add one.

webpack-cli 1.3.2

### Config

https://gist.github.com/timoxley/bd2abb362b3ead91012581729fb9c108

### Error 

```
✔ Reading webpack config
 ❯ Migrating config from v1 to v2
   → undefined does not match field "value": string | boolean | null | number | RegExp of type Literal
   ✔ loadersTransform
   ✔ resolveTransform
   ✖ removeJsonLoaderTransform
     → undefined does not match field "value": string | boolean | null | number | RegExp of type Literal
     uglifyJsPluginTransform
     loaderOptionsPluginTransform
     bannerPluginTransform
     extractTextPluginTransform
     removeDeprecatedPluginsTransform
✖ ︎Migration aborted due to some errors
{ Error: undefined does not match field "value": string | boolean | null | number | RegExp of type Literal
    at add (/Users/timoxley/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:580:31)
    at /Users/timoxley/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:593:21
    at Array.forEach (native)
    at Function.value [as literal] (/Users/timoxley/.config/yarn/global/node_modules/jscodeshift/node_modules/ast-types/lib/types.js:592:34)
    at removeLoaderByName (/Users/timoxley/.config/yarn/global/node_modules/webpack-cli/lib/transformations/removeJsonLoader/removeJsonLoader.js:21:40)
    at NodePath.getLoadersPropertyPaths.forEach.path (/Users/timoxley/.config/yarn/global/node_modules/webpack-cli/lib/transformations/removeJsonLoader/removeJsonLoader.js:38:21)
    at __paths.forEach (/Users/timoxley/.config/yarn/global/node_modules/jscodeshift/dist/Collection.js:76:36)
    at Array.forEach (native)
    at Collection.forEach (/Users/timoxley/.config/yarn/global/node_modules/jscodeshift/dist/Collection.js:75:18)
    at removeLoaders (/Users/timoxley/.config/yarn/global/node_modules/webpack-cli/lib/transformations/removeJsonLoader/removeJsonLoader.js:38:5)
```

**Comment by ev1stensberg** [comment]
May be fixed in a new version push. CC-ing @okonet to conclude it.

**Comment by okonet** [comment]
Thanks for reporting! I'll check against master.

**Comment by okonet** [comment]
I didn't get this error on master so I guess it was fixed in #150
